src/Polly.Shared/CircuitBreaker/CircuitBreakerPolicyAsync.cs

```diff
@@ -10,7 +10,7 @@ namespace Polly.CircuitBreaker
 {
     internal partial class CircuitBreakerPolicy
     {
-        internal static async Task ImplementationAsync(Func<Task> action, IEnumerable<ExceptionPredicate> shouldRetryPredicates, ICircuitBreakerState breakerState)
+        internal static async Task ImplementationAsync(bool continueOnCapturedContext, Func<Task> action, IEnumerable<ExceptionPredicate> shouldRetryPredicates, ICircuitBreakerState breakerState)
```

Comment:
Request move `bool continueOnCapturedContext` to be last parameter, in `ImplementationAsync`